docs/eso/eso.rst

```diff
@@ -125,110 +155,105 @@ list of available instrument-specific queries can be obtained with the
-        endtime: 00 (00 hrs [UT]), 01 (01 hrs [UT]), 02 (02 hrs [UT]), 03 (03 hrs [UT]), 04 (04 hrs [UT]), 05 (05 hrs [UT]), 06 (06 hrs [UT]), 07 (07 hrs [UT]), 08 (08 hrs [UT]), 09 (09 hrs [UT]), 10 (10 hrs [UT]), 11 (11 hrs [UT]), 12 (12 hrs [UT]), 13 (13 hrs [UT]), 14 (14 hrs [UT]), 15 (15 hrs [UT]), 16 (16 hrs [UT]), 17 (17 hrs [UT]), 18 (18 hrs [UT]), 19 (19 hrs [UT]), 20 (20 hrs [UT]), 21 (21 hrs [UT]), 22 (22 hrs [UT]), 23 (23 hrs [UT]), 24 (24 hrs [UT])
-    [x] prog_id:
-    [ ] prog_type: % (Any), 0 (Normal), 1 (GTO), 2 (DDT), 3 (ToO), 4 (Large), 5 (Short), 6 (Calibration)
-    [ ] obs_mode: % (All modes), s (Service), v (Visitor)
-    [ ] pi_coi:
-        pi_coi_name: PI_only (as PI only), none (as PI or CoI)
-    [ ] prog_title:
-    ...
-
-Only the first two sections, of the parameters accepted by the ``midi`` instrument query,
-are shown in the example above: ``Target Information`` and ``Observation and proposal parameters``.
-
-As stated at the beginning of the help message, the parameters accepted by the query are given just before
-the first ``:`` sign (e.g. ``target``, ``resolver``, ``stime``, ``etime``...). When a parameter is prefixed
-by ``[ ]``, the presence of the associated column in the query result can be controlled.
-
-Note: the instrument query forms can be opened in your web browser directly using the ``open_form`` option of
-the :meth:`~astroquery.eso.EsoClass.query_instrument` method. This should also help with the identification of
-acceptable keywords.
+    INFO:
+    Columns present in the table ist.midi:
+        column_name     datatype    xtype     unit
+    ------------------- -------- ----------- ------
+         access_estsize     long              kbyte
+             access_url     char
+           datalink_url     char
+               date_obs     char
+                    dec   double                deg
+          del_ft_sensor     char
+          del_ft_status     char
+                det_dit    float                  s
+               det_ndit      int
+          dimm_fwhm_avg    float             arcsec
+          dimm_fwhm_rms    float             arcsec
+                 dp_cat     char
+                  dp_id     char
+                    ...      ...
+           release_date     char   timestamp
+               s_region     char adql:REGION
+                    ...      ...
+              telescope     char
+              tpl_expno      int
+                 tpl_id     char
+               tpl_name     char
+               tpl_nexp      int
+              tpl_start     char
+                    utc    float                  s
+
+    Number of records present in the table ist.midi:
+    421764
+     [astroquery.eso.core]
+
+**Note:** for a deeper description of each column, the following query can be issued
+on the ESO **Programmatic Access** website (https://archive.eso.org/programmatic/#TAP):
+
+``select column_name, description from TAP_SCHEMA.columns where table_name = 'ist.midi'``
```

Comment:
is there any reason that same query can't be done from astroquery?  Or is the formatting just nicer on the web?